lib/msal-core/src/Account.ts

```diff
@@ -20,7 +21,7 @@ export class Account {
     homeAccountIdentifier: string;
     userName: string;
     name: string;
-    idToken: Object;
+    claims: QPDict;
```

Comment:
Not sure this is a Query Parameter Dictionary, same type but the name is not right. 
Should we rename QP dictionary to Dict which is a generic <String, String> interface

Reply:
Agree. 👍

Reply:
Is this called claims in other MSAL libraries as well? I would prefer something like idTokenClaims to be more descriptive.

Reply:
We use idTokenClaims down below as well.

Reply:
+1 for changing QPDict to somethign else.

Reply:
Done. Changed to Dict.